libraries/mbed/targets/hal/TARGET_STM/TARGET_STM32F4/serial_api.c

```diff
@@ -479,4 +631,537 @@ void serial_break_clear(serial_t *obj)
 {
 }
 
+//########################################################################################
+
+#if DEVICE_SERIAL_ASYNCH
+
+//----------------------------------------------------------------------------------------
+// LOCAL HELPER FUNCTIONS
+//----------------------------------------------------------------------------------------
+
+/** Configure the TX buffer for an asynchronous write serial transaction
+ *
+ * @param obj       The serial object.
+ * @param tx        The buffer for sending.
+ * @param tx_length The number of words to transmit.
+ */
+void h_serial_tx_buffer_set(serial_t *obj, void *tx, int tx_length, uint8_t width)
```

Comment:
shall be static this function as it's for only this compilation unit, or any other (rx buffer set)